docs/reference/ssl/kubernetes-with-cert-manager.md

```diff
@@ -0,0 +1,197 @@
+# SSL on Kubernetes
+
+The quickest way to get automated SSL/TLS certificates for your project is to use [cert-manager][cert-manager]. In this tutorial, we will deploy OpenFaaS using the [Helm Chart][openfaas-helm], [cert-manager][cert-manager], and [nginx-ingress][nginx-ingress]
```

Comment:
Let's mention that this is for an internet-facing gateway. We have users who work behind firewalls and use internal systems, too.